docs/api.md

```diff
@@ -247,6 +247,33 @@ test('waiting for an Banana to be ready', async () => {
 });
 ```
 
+## `wait`
+
+Defined as:
```

Comment:
Let's just state that this is a convenience sugar over `waitForElement` to let you wait for any async value to get present, not only elements. Plus it's possible to run it without an argument to just await next event loop tick